source/administration/important-upgrade-notes.rst

```diff
@@ -9,6 +9,23 @@ Important Upgrade Notes
 +====================================================+=================================================================================================================================================================+
 | v4.9.0                                             | To improve the production use of Mattermost with Docker, the docker image is now running a as non-root user and listening on port 8000. Please read the         |
 |                                                    | `upgrade instructions <https://github.com/mattermost/mattermost-docker#upgrading-mattermost-to-49>`_ for important changes to existing installations.           |
+|                                                    +-----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Several configuration settings have been migrated to roles in the database and changing their config.json values no longer takes effect. These permissions      |
```

Comment:
Do we want to make any special note of the edit message permissions, since that seems to be causing issues?

Reply:
Added another note